web/client/plugins/manager/Manager.jsx

```diff
@@ -0,0 +1,74 @@
+/**
```

Comment:
We usually put the jsx of plugins in the plugins folder, not in a subfolder. We should not change this, unless we do the same for all the other plugins